spec/requests/api/v3/providers/courses/index_spec.rb

```diff
@@ -123,9 +123,6 @@ def perform_request
             "meta" => {
               "count" => 1,
             },
-            "links"  => {
-              "last" => "/api/v3/recruitment_cycles/2020/providers/#{provider.provider_code}/courses?page%5Bpage%5D=1",
```

Comment:
How come this changed?

Reply:
seems to be a feature of `api-pagination` we has now been removed. If we're not using it i say we should remove it